lib/features/welcome/views/widgets/welcome_button.dart

```diff
@@ -0,0 +1,27 @@
+import 'package:flutter/material.dart';
+import 'package:kahootify/color_consts.dart';
+
+class WelcomeButton extends StatelessWidget {
+  final Function onPressed;
+  final String value;
+
+  const WelcomeButton({Key key, this.onPressed, this.value}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return SizedBox(
+      width: 250,
+      height: 100,
+      child: RaisedButton(
```

Comment:
RaisedButton is DEPRECATED!!!! and android studio told you that for sure! Use elevated button instead!

Reply:
Ready